openedx/core/djangoapps/theming/core.py

```diff
@@ -2,66 +2,80 @@
 Core logic for Comprehensive Theming.
 """
 
+from os.path import basename
+
 from django.conf import settings
 
+from path import Path
+
 import edxmako
 
 
-def comprehensive_theme_changes(theme_dir):
+def get_configuration(path_theme):
```

Comment:
Another generalization. We don't need `theme` in the method name; it's already in the name of the module.

"changes" itself is a bit ambiguous. I know that what the implementation currently does with this result is "change" something with it, but the method itself is simply returning the configuration required. "change" implies a delta, where "configuration" does not.

I also prefer verbs in method names :)